Dockerfile

```diff
@@ -41,5 +41,29 @@ WORKDIR $APP_HOME
 RUN chown node:node . 
```

Comment:
From a security perspective, would it make sense _not_ to have the app files owned by the same user that runs it?